### DISCUSSION TITLE
Add Support To Run EXE For Windows 3.11 ?

### DISCUSSION THREAD

**Discussion by cheznewa** [corbindavenport/cobalt#19] [Ideas]
Hi/Hello Is Possible To Add Support To Run Programs 16 Bit For Windows 3.11?

Without Install The Real Windows 3.11

**Comment by corbindavenport** [comment]
Not without Windows, no. [Wine](https://www.winehq.org/) could probably handle them if you're looking for a non-Microsoft solution.
